common.sh

```diff
@@ -41,6 +43,11 @@ function set_build_vars {
     elif [[ "$PLATFORM" == pizdaint ]]; then
         export CONDUIT="${CONDUIT:-aries}"
         export GPU_ARCH=pascal
+    elif [[ "$PLATFORM" == perlmutter ]]; then
+         export CONDUIT=ofi-slingshot11
+         export NUM_NICS=4
```

Comment:
NUM_NICS is no longer necessary (I'm just assuming a max of 8; in the words of Bill Gates, that ought to be enough for anybody)

Reply:
https://github.com/nv-legate/quickstart/pull/7/commits/8904f10df2b5a799f703cd26aae8302f3d79f242